src/content-tags-drawer/ContentTagsDropDownSelector.jsx

```diff
@@ -43,6 +54,9 @@ const ContentTagsDropDownSelector = ({
               className="taxonomy-tags-selectable-box"
               aria-label={`${taxonomyTag.value} checkbox`}
               data-selectable-box="taxonomy-tags"
+              value={[...lineage, encodeURIComponent(taxonomyTag.value)].join(',')}
```

Comment:
I'm not sure if you really need to put the `lineage` into the DOM at all, but if you do, please join the values with `\t` (TAB character) because it's reserved for that purpose. Tags are allowed to have "," in their values which will break this if you use `,` as a separator.

Reply:
I see. Fine either way then.